src/databricks/labs/ucx/source_code/graph.py

```diff
@@ -161,7 +162,10 @@ def build_graph_from_python_source(self, python_code: str) -> list[DependencyPro
         linter = ASTLinter.parse(python_code)
         syspath_changes = PythonLinter.list_sys_path_changes(linter)
         run_calls = PythonLinter.list_dbutils_notebook_run_calls(linter)
-        import_sources = PythonLinter.list_import_sources(linter)
+        import_sources_and_problems = PythonLinter.list_import_sources(linter, DependencyProblem)
+        import_sources = import_sources_and_problems[0]
+        import_problems = import_sources_and_problems[1]
```

Comment:
```suggestion
        sources, problems = PythonLinter.list_import_sources(linter, DependencyProblem)
```
destructure return tuples, don't access them raw.

Reply:
done